pvlib/pvsystem.py

```diff
@@ -1887,6 +1887,82 @@ def sapm_celltemp(poa_global, wind_speed, temp_air,
     return pd.DataFrame({'temp_cell': temp_cell, 'temp_module': temp_module})
 
 
+def pvsyst_celltemp(
+    poa_global,
+    wind_speed,
+    temp_air,
+    eta_m=0.1,
+    alpha_absorption=0.9,
+    temp_model="freestanding",
+):
+    """
+    Calculate PVSyst cell temperature.
+
+    Parameters
+    ------------
+    poa_global : float or Series
+        Total incident irradiance in W/m^2.
+
+    wind_speed : float or Series
+        Wind speed in m/s at a height of 10 meters.
+
+    temp_air : float or Series
+        Ambient dry bulb temperature in degrees C.
+
+    eta_m : numeric
+        PV module efficiency as a fraction
+
+    alpha_absorption : float
+        Absorption coefficient, default is 0.9
+
+    temp_model : string, list, or dict, default 'freestanding'
+        Model to be used.
+
+        If string, can be:
+
+            * 'freestanding' (default)
+            * 'insulated'
+
+        If tuple/list, supply parameters in the following order:
+
+            * natural_convenction_coeff : float
+                Natural convection coefficient. Freestanding default is 29,
+                fully insulated arrays is 15. Recommended values are between
+                23.5 and 26.5.
+
+            * forced_convection_coeff : float
+                Forced convection coefficient, default is 0. Recommended values
+                are between 6.25 and 7.68.
+    Returns
+    --------
+    temp_cell : numeric or Series
+        Cell temperature in degrees Celsius
+    """
```

Comment:
Do we have a reference to point to?

Reply:
I will include the reference to Faiman. I quickly looked in the online docs for PVsyst and didn't find a good reference there.

Reply:
I'd point to the PVsyst user manual, "Array thermal losses" page.  Hard to give a URL as a reference, though.  I'd also reference the Faiman paper (take it off the pvpmc.org site). But the PVsyst model is different from Faiman, including module efficiency.

Reply:
Awesome, thanks @cwhanse for the suggestions. Both will be included in the next commit.